src/util.cc

```diff
@@ -481,6 +481,34 @@ string StripAnsiEscapeCodes(const string& in) {
 
 int GetProcessorCount() {
 #ifdef _WIN32
+  // Need to use GetLogicalProcessorInformationEx to get real core count on
+  // machines with >64 cores. See https://stackoverflow.com/a/31209344/21475
```

Comment:
Can you put your code in `#if !defined(_WIN64)`? That makes it clear, that it's only needed for a 32 bit binary.

Reply:
Done! Sorry for the delay